lib/components/util/dropdown.tsx

```diff
@@ -169,7 +173,7 @@ export const Dropdown = ({
         tabIndex={0}
         title={label}
       >
-        {name}
+        <NameWrapper>{name}</NameWrapper>
```

Comment:
Are we expecting this to be styled in the future? Why not just make it a `span`?